cpp/tests/CMakeLists.txt

```diff
@@ -333,7 +335,33 @@ set(EXPERIMENTAL_KATZ_CENTRALITY_TEST_SRCS
     "${CMAKE_SOURCE_DIR}/../thirdparty/mmio/mmio.c"
     "${CMAKE_CURRENT_SOURCE_DIR}/experimental/katz_centrality_test.cpp")
 
-ConfigureTest(EXPERIMENTAL_KATZ_CENTRALITY_TEST "${EXPERIMENTAL_KATZ_CENTRALITY_TEST_SRCS}" "")
+ConfigureTest(EXPERIMENTAL_KATZ_CENTRALITY_TEST "${EXPERIMENTAL_KATZ_CENTRALITY_TEST_SRCS}" "" "")
+
+
+###################################################################################################
+# - MG tests --------------------------------------------------------------------------------------
+if(BUILD_CUGRAPH_MG_TESTS)
+    if(MPI_CXX_FOUND)
+        set(MG_EXTRA_INCLUDE_DIRS
+            ${MPI_CXX_INCLUDE_PATH})
+        set(MG_EXTRA_LIBS
+            ${MPI_CXX_LIBRARIES})
```

Comment:
This was easier since I was able to copy cuML's configuration, but I'll look into `mpic++` since that sounds like it might be a better approach.

Reply:
I'm still looking into this, but would we want `mpic++` to be used as the compiler for non-MPI binaries in the build as well (`libcugraph.so`, SG tests, etc.) when MPI is enabled for the MG tests? I'm looking at cmake docs and it doesn't seem common to use different compilers for individual targets, but I might just be missing it.

Reply:
Chatted with @seunghwak offline: it looks like the most standard approach to build a target with MPI in modern cmake is to simply pass `MPI::MPI_C` as an additional library to the individual target(s), like so: `target_link_libraries(MG_PAGERANK_TEST PRIVATE MPI::MPI_C)`.

This will be done in the next commit to this PR.